src/main/java/org/jenkinsci/plugins/xunit/threshold/FailedThreshold.java

```diff
@@ -78,4 +95,139 @@ public boolean isValidThreshold(double threshold, double value) {
         return value <= threshold;
     }
 
+    private int getQuarantined(XUnitLog log, TestResult testResultAction, FilePath workspace) {
+
+        if (workspace == null)
+            return 0;
+
+        String quarantinedTestsFileName = "quarantined-tests.json";
```

Comment:
Define as static in the class so it's easy to find and update usages
```suggestion
        private static final String QUARANTINED_TEST_FILE = "quarantined-tests.json";
```